cmd/jaeger/internal/integration/cassandra_test.go

```diff
@@ -4,15 +4,85 @@
 package integration
 
 import (
+	"os"
+	"path/filepath"
 	"testing"
 
+	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v3"
+
 	"github.com/jaegertracing/jaeger/plugin/storage/integration"
 )
 
+func addCreateSchemaConfig(t *testing.T, configFile string) string {
```

Comment:
wait, I think this is overkill. There are two parts to this:
  * the config must contain the parameters needed to create a schema. Can we populate all those parameters in DefaultConfig() function? If not, it's fine to include them in the main config file "../../config-cassandra.yaml", they can be ignored for half of the runs.
  * You need to vary the CreateSchema property to be true/false. This can be achieved by encoding the property in the config with environment variable:
```
  schema:
    create: ${env:CASSANDRA_CREATE_SCHEMA:-true}
```

Reply:
Done.